app/views/users/edit.html.haml

```diff
@@ -8,7 +8,7 @@
           .user-name_image.row
             .user-name.col-md-10
               = f.text_field :name, placeholder: "Username", value: "#{ @user.name }"
-            .user-image.col-md-2.user_avatar_preview
+            .user-image.col-md-2.user_avatar_preview{style: "background-image: url(#{@user.avatar});"}
```

Comment:
Line is too long. [102/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace